### ISSUE TITLE
TLL Moho Geothermal Eco Balance

### ISSUE THREAD

**Issue by ghost** [techannihilation/TA#340]
Lets do some math:
_________________________________________________________
**Tll Moho Geothermal Foundry**

 Metal 6810m
 Energy 30200e

Total cost 6810m + 30200e x (16m/1000e) = 7293.2m 
BuildTime 160000
Income : 35m and +3000/-2500e (remaining energy 500e)

**In conclusion cost of  7293.2m we have +35m and +500e, buildTime 160000**
_________________________________________________________

**Arm Moho Geothermal Foundry**

Metal 5490m
Energy 70390e

Total Geo cost 5490m + 70390e x (16m/1000e)  = 6616,24m
Buildtime 120000
To compare to Tll Geo we need build 2,73 x armmmkr, cost (1009m, 42930e, build 40000)
Total MM cost 2,73 x 1009m + 2,73 x 42930e x 0,016 = 3441,45m
MM Build time 2,73 x 40000 = 109200

Total cost 6616,24m+ 3441,45m = 10057,69m
Build time 109200 + 120000 = 229200
Total Income: 2,73 x 12,8m = 35m and +3000/-2184 (remaining energy 816e )

**In conclusion cost of  10057,69m we have +35m and 816e, buildTime 229200**
_________________________________________________________

**Comment by ksilversurfer** [comment]
So I think we should nerf Tll moho geo a bit.
Suggest values pls.

**Comment by ksilversurfer** [comment]
to be continued :D

**Comment by skymyj** [comment]
Can you continue ? 
I think, we should increase again the Metal cost by 10% and energy cost by 50% for TLL.
